gflownet/gflownet.py

```diff
@@ -202,7 +202,7 @@ def sample_actions(
         sampling_method: Optional[str] = "policy",
         backward: Optional[bool] = False,
         temperature: Optional[float] = 1.0,
-        random_action_prob: Optional[float] = 0.0,
+        random_action_prob: Optional[float] = None,
```

Comment:
Why is this even necessary? Is manually passing `random_action_prob` ever used? (I tried to quickly search for it on my branch and I couldn't find any such cases, but maybe I missed something) And if not, maybe just remove this argument?

Reply:
Yeah true it's not currently used I think. I thought it could be useful in certain situations. The one I am thinking is that we may want to have a schedule on the random_action_prob.

Also if we remove that argument, we should remove the temperature too (for consistency) and things would change significantly.

Reply:
I am not sure if removing the temperature would be a significant change, but I see your point about having a scheduler. I am borderline on whether this should be kept for now or not, it's not a big difference either way; approving.

Reply:
Thanks for the review!